.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
  - INTEGRATION=true ES_VERSION=1.7.6 TEST_DEBUG=true
  - INTEGRATION=true ES_VERSION=2.4.4 TEST_DEBUG=true
  - INTEGRATION=true ES_VERSION=5.5.0 TEST_DEBUG=true
- - INTEGRATION=true ES_VERSION=master TEST_DEBUG=true
+ - INTEGRATION=true ES_VERSION=6.x TEST_DEBUG=true
```

Comment:
Yes. I don't think the 7.x branch of this plugin is intended to be tested against Logstash 7.0, which I assume will only work with the 8.x branch.